bin/compile

```diff
@@ -33,6 +35,7 @@
 #
 # Daniel Weibel <daniel@weibeld.net> March 2015 - April 2018
 #------------------------------------------------------------------------------#
+set -e
```

Comment:
Please add `set -x` too. I think it's good to know which command caused the error.

Reply:
Oh, shit, I pushed this in the wrong repo.